### DISCUSSION TITLE
Docker installation SElinux

### DISCUSSION THREAD

**Discussion by gilbertf** [pretix/pretix#4233] [Show and tell]
### Problem and impact

Installing pretix following https://docs.pretix.eu/en/latest/admin/installation/docker_smallscale.html failed on Fedora 39 with SElinux (default installtion) blocking access (permission denied) to /data and redis.sock.

This might likely apply to CentOs and Redhat as well.

### Expected behaviour

1. Update documentation (or add remark) to fix /data access problem by changing systemd pretix.service ExecStart entry from "-v /var/pretix-data:/data" to "-v /var/pretix-data:/data:z"

2. Update documentation (or add remark) to update SElinux rules to accept redis.sock access: "grep 'redis.sock' /var/log/audit/audit.log | audit2allow -M redis-socket; semodule -i redis-socket.pp"

**Comment by raphaelm** [comment]
> Update documentation (or add remark) to fix /data access problem by changing systemd pretix.service ExecStart entry from "-v /var/pretix-data:/data" to "-v /var/pretix-data:/data:z"

That appears to make sense.

> Update documentation (or add remark) to update SElinux rules to accept redis.sock access: "grep 'redis.sock' /var/log/audit/audit.log | audit2allow -M redis-socket; semodule -i redis-socket.pp"

Is that really a best practice on how to do that? Sounds more like a ``curl|bash`` approach that I wouldn't want to recommend in public docs.
